test/index.test.ts

```diff
@@ -181,4 +181,17 @@ describe('fp-ts-ramda', () => {
       )
     );
   });
+  it('anyPass', () => {
+    const odd = (n: number) => n % 2 !== 0;
+    const lt20 = (n: number) => n < 20;
+    const gt5 = (n: number) => n > 5;
+    fc.assert(
+      fc.property(
+        fc.integer(),
+        num =>
+          eqBoolean.equals(R.anyPass([odd, lt20, gt5])(num), FR.anyPass([odd, lt20, gt5])(num)) &&
+          eqBoolean.equals(R.anyPass([])(num), FR.anyPass([])(num))
```

Comment:
As explained in #24 , I think is better if we always use the non-curried form in tests for the `FR.` implementations, so as to spot potential "manual currying" runtime issues
```suggestion
          eqBoolean.equals(R.anyPass([odd, lt20, gt5])(num), FR.anyPass([odd, lt20, gt5], num)) &&
          eqBoolean.equals(R.anyPass([])(num), FR.anyPass([], num))
```